Package.swift

```diff
@@ -56,6 +56,7 @@ if shouldIncludeDocCPlugin {
 // See https://github.com/RevenueCat/purchases-ios/pull/2989
 // #if os(visionOS) can't really be used in Xcode 13, so we use this instead.
 let visionOSSetting: SwiftSetting = .define("VISION_OS", .when(platforms: [.visionOS]))
+let virtualCurrenciesSetting: SwiftSetting = .define("ENABLE_VIRTUAL_CURRENCIES")
```

Comment:
We needed to enable the `ENABLE_VIRTUAL_CURRENCIES` compiler flag to access the `VirtualCurrencyInfo` struct. This will be removed when we eventually merge into main